packages/mongodb-chatbot-ui/src/ModalView.tsx

```diff
@@ -143,6 +143,7 @@ export function ModalView(props: ModalViewProps) {
       size="large"
       initialFocus={`#${ActiveInputBarId}`}
       shouldClose={shouldClose}
+      placeholder=""
```

Comment:
@nlarew when i updated dependencies for the project (deleting node_modules + package-lock.json), i had to add the `placeholder` property here. this seems to work w/o visual regression based on manual testing, but could you take a look at this before merge?

Reply:
Strange what error do you get if you remove it? Modal doesn't take a `placeholder` prop (I'm not really sure what that would even mean)

I can't reproduce but sounds like something wacky is happening

Reply:
try to 1) check out this branch 2) `rm -rf node_modules package-lock.json` 3) npm i lerna && npm run bootstrap

Reply:
Fixed (I think) in the latest commit by upgrading the version of `@leafygreen-ui/modal`